docs/admin/install/eventing/install-eventing-with-yaml.md

```diff
@@ -53,143 +52,148 @@ Follow the procedure for the channel of your choice:
         ```
 
         !!! tip
-            To learn more, try the [Apache Kafka channel sample](../../../eventing/samples/kafka/channel/README.md).
+            To learn more, try the [Apache Kafka Channel sample](../../../eventing/samples/kafka/channel/README.md).
 
 === "Google Cloud Pub/Sub Channel"
 
-    1. Install the Google Cloud Pub/Sub channel:
+    * Install the Google Cloud Pub/Sub Channel by running the command:
 
         ```bash
-        # This installs both the Channel and the GCP Sources.
         kubectl apply -f {{ artifact(org="google",repo="knative-gcp",file="cloud-run-events.yaml")}}
         ```
 
+        This command installs both the Channel and the GCP Sources.
+
         !!! tip
             To learn more, try the [Google Cloud Pub/Sub channel sample](https://github.com/google/knative-gcp/blob/master/docs/examples/channel/README.md).
 
 
 === "In-Memory (standalone)"
 
-    The following command installs an implementation of channel that runs in-memory.
-    This implementation is nice because it is simple and standalone, but it is unsuitable for production use cases.
+    !!! warning
+        This simple standalone implementation runs in-memory and is unsuitable for production-use cases.
```

Comment:
```suggestion
        This simple standalone implementation runs in-memory and is not suitable for production use cases.
```

Reply:
@abrennan89, why "not suitable" over "unsuitable"? 
"Unsuitable" seems the safer choice to me because the reader's eye could skip "not" whereas it's much harder to read "unsuitable" as "suitable".

Google say much the same thing for contractions:
"One reason that such contractions are useful is that it's sometimes easy for a reader to miss the word not, whereas it's harder to misread don't as do."
https://developers.google.com/style/contractions

Reply:
Whatever you think, it was just a suggestion